src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  --active-color--: #00acdc;
+}
+
+* {
+  border: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  box-sizing: border-box;
```

Comment:
1. It is better to set global font styles in `html` or `:root` selectors
2. `box-sizing` in `*`